### PR TITLE
shellinabox: 2.14 -> 2.16

### PR DESCRIPTION
**Pull request by tomberek**

### CODE REVIEW

**Comment by geerds** [comment]
Please reword your commit message according to our [guidelines](https://hydra.nixos.org/build/24422016/download/1/nixpkgs/manual.html#chap-submitting-changes) (e.g. `shellinabox: 2.14 -> 2.16`)

/cc maintainers @tomberek @lihop

**Comment by tomberek** [comment]
Updated to use unofficial fork on github. Original author and group are unresponsive. The fork is far ahead on updates,  bug fixes, and changes.

File: pkgs/servers/shellinabox/default.nix [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+{ stdenv, fetchgit, autoconf, automake, libtool, pam, openssl, openssh, shadow, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  version = "2.14";
+  version = "2.16";
   name = "shellinabox-${version}";
 
-  src = fetchurl {
-    url = "https://shellinabox.googlecode.com/files/shellinabox-${version}.tar.gz";
-    sha1 = "9e01f58c68cb53211b83d0f02e676e0d50deb781";
```
Comment (lethalman) [inline pkgs/servers/shellinabox/default.nix]: Use fetchFromGitHub please, fetchgit is non deterministic in most cases with github.

**Comment by lihop** [comment]
:+1: I'm very much in favor of using the GitHub fork of shellinabox. This will allow, for example, using fail2ban with the shellinabox module. I've tested this locally and it works great with the service I added in  1eb50ebbf26d0cedc6677dabc78ba5253caf1fa1.

**Comment by tomberek** [comment]
@lethalman: Wasn't aware of fetchFromGitHub.

@lihop: thanks, I've been making my own service module, I'll take a look at `1eb50eb`

**Comment by jagajaga** [comment]
Please don't forget about @geerds's suggestion.

**Comment by pSub** [comment]
@tomberek If you rename the commit message accordingly I will merge this PR.

**Comment by tomberek** [comment]
@pSub I kept thinking they meant the PR title, not the commit name.

Fixed.

**Pull request merged**